### DISCUSSION TITLE
Automatic push to talk & Random sound keybind

### DISCUSSION THREAD

**Discussion by Sahukei** [Tom4nt/Mega-Soundboard#38] [Ideas]
When playing a sound, can it hold down an assignable button for push to talk (for use in Discord/other PTT applications)
Also if you can, a keybind to play a random sound from the current playlist

**Comment by Tom4nt** [comment]
This is a planned feature (see #25), but now it has higher priority since it was requested twice.
Random sounds are also planned and are very easy to implement.

Thanks for contributing!

**Comment by Tom4nt** [comment]
Automatic push to talk done in 46df8f3. Will be released in the next version. Random sounds will be implemented soon.

**Comment by Tom4nt** [comment]
Random sounds done. (b560deff7e228e3f065981d41b2864f65cbe74d7)
